src/Text/Pandoc/Writers/Markdown/Inline.hs

```diff
@@ -116,9 +124,62 @@ attrsToMarkdown attribs = braces $ hsep [attribId, attribClasses, attribKeys]
               escAttrChar '\\' = literal "\\\\"
               escAttrChar c    = literal $ T.singleton c
 
+attrsToMarkua:: Attr -> Doc Text
+attrsToMarkua attributes
+    | null list = empty
+    | length list == 1 = braces $ head list
+    |  otherwise = braces $ hcat (head list : (map (\x -> ", " <> x)
+                                                   (tail list)))
+        where attrId = case attributes of
+                        ("",_,_) -> []
+                        (i,_,_)  -> [literal $ T.append "id: " i]
+              -- all non explicit (key,value) attributes besides id are getting
+              -- a default class key to be Markua conform
+              attrClasses = case attributes of
+                             (_,[],_) -> []
+                             (_,classes,_) -> map (escAttr . ("class: " <>))
+                                classes
+              attrKeyValues = case attributes of
+                               (_,_,[]) -> []
+                               (_,_,keyvalues) -> map (\(k,v) -> escAttr k
+                                              <> ": " <>
+                                              escAttr v)
+                                              (filter (/=(T.empty,T.empty))
+                                              (map preprocessKeyValues
+                                                   keyvalues))
+              escAttr          = mconcat . map escAttrChar . T.unpack
+              escAttrChar '"'  = literal "\""
+              escAttrChar c    = literal $ T.singleton c
+
+              list = concat [attrId, attrClasses, attrKeyValues]
+
+              -- if attribute key is alt, caption, title then content
+              -- gets wrapped inside quotes
+              -- if some value is automatically set to "fig:", then this
+              -- attribute gets removed
+              preprocessKeyValues :: (Text, Text) -> (Text, Text)
+              preprocessKeyValues (key,value)
+                 | value == "fig:" = (T.empty,T.empty)
+                 | T.head value == '\"' = (key, value)
+                 | key == "alt" ||
+                   key == "caption" = (key, T.cons '\"' (T.snoc value '\"'))
```

Comment:
Even though this might be a tiny bit more efficient, for code clarity I think you can't beat `"\"" <> value <> "\""`.
Or even better define your own `inquotes`.

Reply:
We changed it accordingly